### DISCUSSION TITLE
how to support mod,

### DISCUSSION THREAD

**Discussion by tinklesdinkles** [zippity21/Stardew_Valley_Regression_Mod#132] [Q&A]
i enjoy this mod a lot and would love to support to it in some way, but am not really sure how to code anything for star dew. 
are there any guides or a good means to support this project?

**Comment by TikkuTokku** [comment]
A couple of us have been working to finish out all the dialogue for the remaining characters. I've personally taken' a pause from writing as I work on IRL stuff, but another of us has been chipping away at a few characters. If you don't do coding, that's the best way to help.
